module/Decision/src/Decision/Service/Member.php

```diff
@@ -72,6 +72,24 @@ public function getMembershipInfo($lidnr = null)
         ];
     }
 
+    /**
+     * Returns is the member is active
+     *
+     * An active member is a member that is part of at least one organ or is part of
+     * the board.
+     *
+     * @param MemberModel $member
+     * @return bool
+     */
+
+    public function isActiveMember(MemberModel $member)
```

Comment:
I don't know how correct this method is but it gets the job done for presenting the correct content. We should probably rely on proper ACL's when making changes.